From 640c2b3c0c72d5194d422ff7dda086c4533fba00 Mon Sep 17 00:00:00 2001
From: Yegor Yefremov <yegorslists@googlemail.com>
Date: Mon, 13 Jan 2020 14:08:44 +0100
Subject: [PATCH] CMake: reorder Python detection

PythonInterp should be detected before PythonLibs otherwise
CMake exits with the following message:

Could NOT find PythonLibs (missing: PYTHON_LIBRARIES PYTHON_INCLUDE_DIRS)

Signed-off-by: Yegor Yefremov <yegorslists@googlemail.com>
---
 python/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/python/CMakeLists.txt b/python/CMakeLists.txt
index 90811ca..368bb7e 100644
--- a/python/CMakeLists.txt
+++ b/python/CMakeLists.txt
@@ -12,8 +12,8 @@ if ( PYTHON_BINDINGS )
       set ( SWIG_FOUND TRUE )
     endif ()
   endif ()
-  find_package ( PythonLibs )
   find_package ( PythonInterp )
+  find_package ( PythonLibs )
 endif ()
 
 if ( SWIG_FOUND AND PYTHONLIBS_FOUND AND PYTHONINTERP_FOUND )
-- 
2.17.0

